Princeton-CDH/cdh-ansible
# Shxco qa playbook and django role revisions

updates to shxco_qa playbook needed for deploy, including:
- include solr_collection role in playbook
- make sure apache fonts are configured correctly
- enable test twitterbot cron job (test twitterbot credentials in qa vault)
- project-specific local settings
- make it possible to customize apache site name and configure so it is not mep

related updates to django role, including:
- default solr config in local settings template
- default logging config and logging path variable for local settings template
- readme
